packages/editor-ui/src/stores/posthog.ts

```diff
@@ -132,12 +133,16 @@ export const usePostHog = defineStore('posthog', () => {
 			window.posthog?.onFeatureFlags?.((keys: string[], map: FeatureFlags) => {
 				featureFlags.value = map;
 				addExperimentOverrides();
-				trackExperiments(map);
+				trackExperimentsDebounced(map);
 			});
 		}
 	};
 
-	const trackExperiments = debounce((featureFlags: FeatureFlags) => {
+	const trackExperiments = (featureFlags: FeatureFlags) => {
+		EXPERIMENTS_TO_TRACK.forEach((name) => trackExperiment(featureFlags, name));
+	};
+
+	const trackExperimentsDebounced = debounce((featureFlags: FeatureFlags) => {
```

Comment:
something is off.. not sure what broke here.. maybe it was this change.. sorry if so.. but `User is part of experiment` telemetry event is not firing anymore

Reply:
maybe a debounce or `setTimeout` is necessary here to get this event 
<img width="591" alt="Bildschirmfoto 2023-04-18 um 15 31 47" src="https://user-images.githubusercontent.com/4711238/232793327-98f22fb9-4c7a-418a-8c2b-03ac24f52783.png">